### PR TITLE
Quest initialization

### PR DESCRIPTION
**Pull request by MuteTiefling**
Following the same basic structure as previous packs. New FTB Quests defaults all text to be lang based, so putting this up for translation should be very simple. 

- [ ] Getting Started
- [ ] Discovery
  - [ ] Building
  - [x] Logistics (Needs rewards)
  - [ ] Redstone
  - [x] Storage (Needs rewards)
- [ ] Learning - Magic
  - [ ] Occultism
  - [ ] Theurgy
- [ ] Learning - Technology
  - [x] Applied Energistics 2 (Getting Started complete)
  - [x] Mekanism
  - [x] Modern Industrialization (Complete through MV)

### CODE REVIEW

File: command_history.txt
Comment (NielsPilgaard) [inline command_history.txt:1]: I think we need to gitignore this file :D

File: config/ftbquests/quests/lang/en_us.snbt
Comment (NielsPilgaard) [inline config/ftbquests/quests/lang/en_us.snbt:1]: Did you do this or is it out of the box now? :o

Comment (MuteTiefling) [reply config/ftbquests/quests/lang/en_us.snbt:1]: Out of the box! Its a very nice feature :D

File: command_history.txt
Comment (MuteTiefling) [reply command_history.txt:1]: ah, yep. damn sneaky file

**Comment by MuteTiefling** [comment]
@NielsPilgaard I'm going to attempt to leave any given commit in a 'workable' state for quests. That is to say, feel free to merge them at any time as you produce new versions. The quests can go out as they are and will get updated over time.

**Pull request merged**